tests/unit-tests/helpers/fileProcessor-test.js

```diff
@@ -0,0 +1,32 @@
+const rewire = require('rewire'),
```

Comment:
add:
1. integration test that fails to get file => test should fail
2. unit test of model/runTest where `writeFileToLocalFile` fails => test should fail

Reply:
added